mgmtsystem_nonconformity/views/mgmtsystem_cause.xml

```diff
@@ -0,0 +1,62 @@
+<openerp>
+    <data>
+        <!-- Cause -->
+
+        <record id="view_mgmtsystem_nonconformity_cause_form" model="ir.ui.view">
+            <field name="name">mgmtsystem.nonconformity.cause.form</field>
+            <field name="type">form</field>
+            <field name="model">mgmtsystem.nonconformity.cause</field>
+            <field name="arch" type="xml">
+                <form string="Causes of nonconformity">
+                <field name="name"/>
```

Comment:
I think you need a `<group>` element around the fields for the labels to be displayed.